### PR TITLE
Update Ecosystem and Developer Docs for Witnet Oracle

### PR DESCRIPTION
**Pull request by parodyBit**
## Overview

Update the ecosystem listing for Witnet as well as the developer documentation for price feeds.

## Changes

- update ecosystem project list for Witnet
- replace witnet.svg
- update price-feeds.md developer documentation for Witnet

### CODE REVIEW

File: packages/boba/gateway/src/containers/ecosystem/project.list.js [hunk 1]
```diff
@@ -148,15 +148,15 @@ export const projectList = [
...
-    "title": "WitNet",
+    "title": "Witnet Oracle",
     "canLaunch": false,
     "link": "https://witnet.io/",
     "telegram": "https://t.me/witnetio",
     "twitter": "https://twitter.com/witnet_io",
-    "discord": "https://discord.gg/X4uurfP",
-    "type": "tools",
+    "discord": "https://discord.gg/witnet",
+    "type": "tool",
```
Comment (sk-enya) [inline packages/boba/gateway/src/containers/ecosystem/project.list.js:158]: Please revert type to `Tools` as this parameter defines the category to show on gateway.

Comment (parodyBit) [reply packages/boba/gateway/src/containers/ecosystem/project.list.js:158]: gm-

It currently is `tools` and does not show on https://boba.network/ecosystem/
which prompted this pr.

the projects that show correctly have the type listed as `"tool"` as described here https://github.com/bobanetwork/boba/blob/develop/packages/boba/gateway/src/containers/ecosystem/project.list.js#L38-L43

please confirm and i will revert it.

-thanks

Comment (sk-enya) [reply packages/boba/gateway/src/containers/ecosystem/project.list.js:158]: yeah, it's correct!

**Review by InoMurko — APPROVED** [review]

**Comment by boyuan-chen** [comment]
Should we merge this to the docs branch?

**Review by sk-enya — APPROVED** [review]

**Comment by sk-enya** [comment]
> Should we merge this to the docs branch?

@Boyuan-Chen https://github.com/bobanetwork/boba/issues/333 created Issues here will address that separately.

**Review by boyuan-chen — APPROVED** [review]

**Comment by InoMurko** [comment]
@java-spaces can you add this to our docs.boba.network please? thanks!

**Pull request merged**